brozzler/__init__.py

```diff
@@ -19,9 +19,9 @@
 
 import logging
 import structlog
-from pkg_resources import get_distribution as _get_distribution
+from importlib.metadata import version as _version
 
-__version__ = _get_distribution("brozzler").version
+__version__ = _version("brozzler")
```

Comment:
I should note that, like the f-strings that we've added recently, this sets a floor on our Python version - importlib was introduced in 3.8. Our pyproject.toml already declared 3.8 as our minimum Python version though.